src/main/java/gov/cdc/izgateway/logging/event/Health.java

```diff
@@ -85,7 +85,11 @@ public class Health {
 	@JsonProperty 
 	@Schema(description="Host name as known by the operating system")
 	private String hostname;
-
+	
+	@JsonProperty
+	@Schema(description="The database in use")
+	private String database;
```

Comment:
Is this change related to dynamodb? Is it intended for this release?

Reply:
Yes, it supports DynamoDb, but is harmless for 2.1.6